src/main/java/com/puppycrawl/tools/checkstyle/checks/indentation/MemberDefHandler.java

```diff
@@ -77,10 +77,9 @@ protected void checkModifiers() {
      */
     private void checkType() {
         final DetailAST type = getMainAst().findFirstToken(TokenTypes.TYPE);
-        final DetailAST ident = AbstractExpressionHandler.getFirstToken(type);
```

Comment:
This was the only usage of `AbstractExpressionHandler.getFirstToken()` in the project, so I have removed this method.